instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/utils/http/MockDirectionsRequestHandler.kt

```diff
@@ -30,7 +38,8 @@ data class MockDirectionsRequestHandler(
         }
 
         return if (request.path!!.startsWith(prefix)) {
-            MockResponse().setBody(jsonResponseModifier(jsonResponse))
+            Thread.sleep(serverDelayMs)
```

Comment:
We already have a `DelayedResponseModifier`. If I'm not mistaken, you can pass an original modifier to create a modifier chain.